src/components/Share/Share.js

```diff
@@ -48,26 +54,36 @@ const renderConf = (conf, t) => (
 );
 
 const Share = ({ appBaseUrl }) => {
+  const activeTopic = useSelector(state => state.app.activeTopic);
+  const language = useSelector(state => state.app.language);
   const map = useSelector(state => state.app.map);
   const { t } = useTranslation();
-  const config = [...socialShareConfig];
+  const duplicateConfig = JSON.parse(JSON.stringify(socialShareConfig));
```

Comment:
As @oterral already wrote: Please don't duplicate the config using `JSON.parse` and `JSON.stringify`.
There are other possibilities to replace the placeholders instantly, e.g. by using a helper function.

`<a href={replaceParams(conf.url)} />` etc.

Reply:
Ok I replaced with a helperFunction